docs/middleware.md

```diff
@@ -23,11 +23,12 @@ Orka adds by default the following middleware:
 - errorHandler (Handles errors thrown in any middleware added afterwards)
 - [koa-compress](https://www.npmjs.com/package/koa-compress)
 - [koa2-cors](https://www.npmjs.com/package/koa2-cors)
-- addVisitorId (adds a visitorId in ctx.state.visitorId from cookie ctx.visitor.cookie)
+- addVisitorId (adds a visitorId in ctx.state.visitor from cookie ctx.visitor.cookie)
```

Comment:
❓ Is this a fix to the docs or did the implementation change? The first right?

Reply:
Yes, this is a fix to the docs